scripts/resource/README.md

```diff
@@ -31,23 +31,11 @@ poetry run python scripts/resource/class_generator.py --interactive
 
 #### Adding tests
 
-- Generate the resource file the debug file by adding `--debug` flag
-- Replace `Pod` with the kind you want to test
+- Generate the resource file and json file by adding `--add-tests` flag
```

Comment:
No need to explain what we generate, only that this option will add a new test for the given kind